src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,11 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder buffer = new StringBuilder();
```

Comment:
let's give better name for buffer, cause there is StringBuffer also so it might be confusing

Reply:
done, i renamed variable